### ISSUE TITLE
Recommendations are not my own?

### ISSUE THREAD

**Issue by pannal** [jdf76/plugin.video.youtube#323]
The recommendations I receive are not in any way the same that I receive on YouTube.

Is this by design or an API limitation?

**Comment by anxdpanic** [comment]
This is an API limitation

**Comment by and-hom** [comment]
See here https://developers.google.com/youtube/v3/docs/activities/list 

**home**  boolean       
Note: This parameter has been deprecated.For  requests that set this parameter, the API response contains items  similar to those that a **logged-out** user would see on the YouTube home  page. Note that this parameter can only be used in a properly authorized request.

**Comment by anxdpanic** [comment]
Yes, and the mine does not work, and channelId=<your channel id> returns empty.

**Comment by mattreecebentley** [comment]
That's a shame, it's a nice feature when it works.

**Comment by dromer** [comment]
Imo the recommendations don't make any sense this way. There is nothing 'recommended' about these, it's just whatever youtube wants you to see in your region.

(here that means: a lot of really really bad and annoying content)

It would be one of my main feature requests to get proper recommendations going, but apparently the youtube API is just broken this way.

**Comment by jvisser** [comment]
Is it not possible to do whatever the web UI does? I can see the home page request and in the response see the sections and video id's within using the Chrome debugger. But this would probably be really unstable since it is not in the official/public API. 

Not having this feature is the only reason I'm still using the official app. I like the kodi UI better though, also no ads.

**Comment by hedleyroos** [comment]
This was annoying me as well so I finally got round to creating a PR. https://github.com/jdf76/plugin.video.youtube/pull/784
